confgenerator/self_logs.go

```diff
@@ -55,6 +56,40 @@ func healthChecksLogsPath() string {
 	return path.Join("${logs_dir}", "health-checks.log")
 }
 
+type LoggingProcessorSampleLogs struct {
```

Comment:
This is bespoke enough that it's fine staying as raw fluent-bit config. I don't think it's ever going to be reused in another processor.

Reply:
Again, I was hoping to actually turn this into an internal logging processor that was used in an actual pipeline…